pkg/installmanager/installmanager_test.go

```diff
@@ -400,62 +399,63 @@ level=info msg="Install complete!"
+level=info msg="Destroying the bootstrap resources..."
+level=info msg="Waiting up to 30m0s for the cluster at https://api.test-cluster.example.com:6443 to initialize..."
+level=info msg="Waiting up to 10m0s for the openshift-console route to be created..."
+level=info msg="Install complete!"
+level=info msg="To access the cluster as the system:admin user when using 'oc', run 'export KUBECONFIG=/output/auth/kubeconfig'"
+level=info msg="Access the OpenShift web-console here: https://console-openshift-console.apps.test-cluster.example.com"
+REDACTED LINE OF OUTPUT`,
 		},
 		{
 			name: "password at start of line",
 			sourceString: `some log line
 password at start of line
 more log`,
-			missingStrings: []string{"password"},
+			expectedString: `some log line
+REDACTED LINE OF OUTPUT
+more log`,
 		},
 		{
 			name: "password in first line",
 			sourceString: `first line password more text
 second line no magic string`,
-			missingStrings: []string{"password"},
+			expectedString: `REDACTED LINE OF OUTPUT
+second line no magic string`,
 		},
 		{
 			name: "password in last line",
 			sourceString: `first line
 last line with password in text`,
-			missingStrings: []string{"password"},
+			expectedString: `first line
+REDACTED LINE OF OUTPUT`,
 		},
 		{
 			name:           "case sensitivity test",
 			sourceString:   `abc PaSsWoRd def`,
-			missingStrings: []string{"PaSsWoRd"},
+			expectedString: `REDACTED LINE OF OUTPUT`,
 		},
 		{
 			name:         "libvirt ssh connection error in console log",
 			sourceString: "Internal error: could not connect to libvirt: virError(Code=38, Domain=7, Message='Cannot recv data: Permission denied, please try again.\\r\\nPermission denied (publickey,gssapi-keyex,gssapi-with-mic,password)",
-			missingStrings: []string{
-				"Permission denied (publickey,gssapi-keyex,gssapi-with-mic,password)",
-			},
-			expectedStrings: []string{
-				"Internal error: could not connect to libvirt: virError(Code=38, Domain=7",
-				"Permission denied, please try again.",
-			},
+			// In addition to redacting the line with "password" the
+			// escaped carriage returns and newlines are unescaped.
+			expectedString: "Internal error: could not connect to libvirt: virError(Code=38, Domain=7, Message='Cannot recv data: Permission denied, please try again.\r\nREDACTED LINE OF OUTPUT",
 		},
 	}
 
 	for _, test := range tests {
 		cleanedString := cleanupLogOutput(test.sourceString)
-
-		for _, testString := range test.missingStrings {
-			assert.False(t, strings.Contains(cleanedString, testString),
-				"testing %v: unexpected string found after cleaning",
-				test.name, testString)
-		}
-
-		for _, testString := range test.expectedStrings {
-			assert.True(t, strings.Contains(cleanedString, testString),
-				"testing %v: expected string %q not found after cleaning: %q became %q",
-				test.name, testString, test.sourceString, cleanedString)
-		}
+		assert.True(t, cleanedString == test.expectedString,
```

Comment:
Use `assert.Equal`. Then you do not need to explicitly add the expected and actual to the message.

```
assert.Equal(t, test.expectedString, cleanedString, "unexpected cleaned string")
```

Reply:
Done